### DISCUSSION TITLE
Receipt types: "read" vs "viewed"

### DISCUSSION THREAD

**Discussion by exquo** [AsamK/signal-cli#948] [Q&A] [answered]
What is the difference between a "read" and a "viewed" receipt? The [official signal docs](https://support.signal.org/hc/en-us/articles/360007059812-Read-Receipts) mention only two receipt types: "delivered" and "read". Skimming the code of a commit that introduced both types (e3752e733adaad6f04c7d525e1b465a4b46474c7), I don't see what functionality the "viewed" receipt types are related to.

**Comment by AsamK** [comment, marked as answer]
"viewed" receipts are used e.g. for voice notes. When the user sees the voice note, a "read" receipt is sent, when the user has listened to the voice note, a "viewed" receipt is sent (displayed as a blue dot in the apps).
I think the same applies for view-once images, but I'm not sure.
